File: lib/scalalib/Bench.scala
package scalalib

object Bench {
	var t0:Long = 0
	var t1:Long = 0

	def benchStart():Unit = {
		t0 = System.currentTimeMillis();
	}

	def benchEnd():Unit = {
		t1 = System.currentTimeMillis();
		val res:Long = t1 - t0;
		System.out.println(res);
	}
}

File: lib/golib/src/bench/bench.go
package bench

import "time"
import "fmt"

var t0, t1 int

func BenchStart(){
	t0 = time.Now().Nanosecond()
}

func BenchEnd(){
	t1 = time.Now().Nanosecond()

	var m0 int = int(t0 / 1e6)
	var m1 int = int(t1 / 1e6)

	var res int = (m1 - m0)
	fmt.Printf("%d",res);
}